tests/singlenode_key_ed25519/singlenode_test.go

```diff
@@ -1,10 +1,7 @@
-//go:build integration
-// +build integration
-
 package test
 
 import (
-	"os"
+	"log"
```

Comment:
same comments :

- I think it should be moved into testing.M function instead and have it ready before everything
- avoid using `log`. we can use `t.LogX` instead
- res should be `output`, not to be confused with the status code